### ISSUE TITLE
Avatar - Finalize sandbox contribution

### ISSUE THREAD

**Issue by KennyAtHPE** [grommet/hpe-design-system#2836]
Sandbox contribution can be found [here](https://www.figma.com/file/Tw2PiwyLS21l20VTXKbQfS/Avatar?node-id=0%3A1)

#### Objectives
The following items must be completed before the Avatar contribution can be published within the Design System Components file

- [x] Recreate Avatar component with Initial, Logo, and Image variants.
       - Remove status indicators, drop shadows are not needed, and create a dark mode variant
- [x] Move to DS Components file
~~- [ ] Deprecate old Avatar components + file~~
- [x] Update [color palette](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=218%3A338) for Initials variant
- [x] Add in missing [Grommet logos](https://icons.grommet.io/?s=logo) to the [icons file](https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons?node-id=261%3A0)
- [ ] Create missing Platform icons (Aditi will reach out to Chris about this)
~~- [ ] Write additional guidance for the DS site (can be broken into a separate ticket if needed)~~
       - [Use cases](https://www.figma.com/file/Tw2PiwyLS21l20VTXKbQfS/Avatar?node-id=402%3A112)

**Comment by KennyAtHPE** [comment]
@vavalos5 please review the component when you have time. You can find it in the assets panel by searching "Sandbox-Test"

We are currently holding off on releasing this publicly right now, but that shouldn't be a blocker for you to use this component.

**Comment by KennyAtHPE** [comment]
Crossed out bullet items and additional next steps are being tracked in #2838

**Comment by KennyAtHPE** [comment]
Tagging @bahriaditi as well

**Comment by vavalos5** [comment]
Looks so good, thanks for doing this Kenny!! 🎉

**Comment by bahriaditi** [comment]
Great work, @KennyAtHPE! This looks good to me.

In terms of next steps, we would need to create tickets for the following:

- Create documentation/ guidelines for use 
- Dev implementation on the Design System site
